dv-export/python/query-changes/README.md

```diff
@@ -115,25 +109,20 @@ The application will fetch the full history for each instance.
 | 2024-02-10 | V2        |           |
 | 2024-02-20 |           | V2        |
 
-Before populating the table, all previous values for `Employee1` and `Employee2` will be removed.
+Before populating the table, the application removes all previous values for `Employee1` and `Employee2`.
 
-### Last Mode Examples:
+### Last Mode Example
 
-In `last` mode, the application identifies analytic object instances that have changed between
-the specified data versions provided by the user.
-For each of these changed instances, it only fetches the most recent version within
-these data versions and appends it to the table.
-The results will be consistent regardless of when the application is run.
+Use `last` mode to fetch the most recent change that occurred for a record between two data versions. In `last` mode, the application appends data to the table without restating the full history. The results are consistent regardless of when you run the sample application.
```

Comment:
I believe explaining what’s happening in several sentences is more effective.

First, the process of finding changed instances is similar to the one in the “### Restate Mode Example”. 
Then we only capture the last state of the analytic object between DV1 and DV2.